### PR TITLE
Create test for character controller

### PR DESCRIPTION
**Pull request by jularmerme**
Create unit tests for Character Controller

### CODE REVIEW

File: src/main/java/com/amplifier/models/Character.java [hunk 1]
```diff
@@ -1,5 +1,47 @@
...
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Data;
+
+@Entity
+@Table(name = "user_character")
+@Data
+@ApiModel(value = "User", description = "This model serves as the basic model for all user characters entity API operations.")
```
Comment (pyaeger) [inline src/main/java/com/amplifier/models/Character.java]: On line 16, shouldn't 
`value = "User" `
be
`value = "Character"`
???

File: src/test/java/com/amplifier/controllers/CharacterControllerTest.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  private MockMvc mockMvc;
+
+  @MockBean
+  CharacterService charService;
+
+  @Test
+  @Order(1)
+  @DisplayName("1. Add New Character")
+  public boolean addNewCharacter() {
+    return true;
```
Comment (FullSol) [inline src/test/java/com/amplifier/controllers/CharacterControllerTest.java]: This should actually be testing the response from the controller. Write as if there is code in the controller (even though there is not). The test will fail, but when the next person comes and does the user controller the test should pass without any changes to the test.

**Review by FullSol — CHANGES_REQUESTED** [review]

**Review by pyaeger — APPROVED** [review]
Thanks for the work!

**Comment by FullSol** [comment]
Please explain what happened here.
There are 39 files that will overwrite whatever is in the dev branch if this is merged in.

I see: [Fix Merge branch conflicts](https://github.com/FullSol/amplifier/pull/56/commits/b29fd61f589eea5b9441ade8ee72853cd43e4a85)

But that would not show the changes like this if it was done with the commands:

```
git switch dev 
git pull 
git switch (your working branch)
git merge dev
```
that would only show the changes you made (not all changes).

I am considering this unsafe to merge until we know how this will affect the dev branch.

**Comment by FullSol** [comment]
Okay, so looking at the commit history you went through and resolved the merge conflict AFTER you did the pull request. That is the job of Patrick and I as a part of the code review. I am reverting the merge and I will review the merge e conflicts.

**Comment by pyaeger** [comment]
Sorry guys. I just don't understand enough to be an effective Gitflow Master.

**Comment by FullSol** [comment]
I looked at the code you wrote Julian and it only needed some minor edits - I've made those. However, GitHub is not giving me the option to review the code before merging it in the way I need to in order to make sure we are not overwriting code everyone else wrote.

You probably did the merge correctly when you reviewed it, but it would be irresponsible for me to merge in the code if I wasn't sure about it.

So, what I am going to do it remove this branch and create a new one. I have your code and I am going to have to complete another pull request. You will have errors, ignore them.

Go ahead and go to dev on your machine and run the `git pull` command. I'll get everything else setup.

@jularmerme

**Review by FullSol — COMMENTED** [review]
see previous comment on PR

**Pull request closed without merging**